CONTRIBUTING.md

```diff
@@ -2,6 +2,36 @@
 
 If you're reading this, you're awesome! Thank you for helping us make this project great and being a part of the Material-UI community. Here are a few guidelines that will help you along the way.
 
+## Git workflow
```

Comment:
```suggestion
## Getting started
```

No need for the jargon here.

Reply:
https://github.com/mui-org/material-ui/pull/17597#issuecomment-536091610 Please check. This section is related only to the git workflow stuff, which is not enough to getting started to contribute. It is intended to describe only the git workflow.